apollo-mpp-utils/api/apollo-mpp-utils.api

```diff
@@ -7,14 +13,10 @@ public final class com/apollographql/apollo3/mpp/Platform : java/lang/Enum {
 }
 
 public final class com/apollographql/apollo3/mpp/UtilsKt {
-	public static final fun assertMainThreadOnNative ()V
 	public static final fun currentThreadId ()Ljava/lang/String;
 	public static final fun currentThreadName ()Ljava/lang/String;
 	public static final fun currentTimeFormatted ()Ljava/lang/String;
 	public static final fun currentTimeMillis ()J
-	public static final fun ensureNeverFrozen (Ljava/lang/Object;)V
-	public static final fun freeze (Ljava/lang/Object;)V
-	public static final fun isFrozen (Ljava/lang/Object;)Z
```

Comment:
Sounds like a breaking change. Mark them deprecated with `Error` level instead?

Reply:
The change is actually due to removing the `actual` implementations - I've added them back but kept `HIDDEN` deprecation level, LMK if that's 👍 with you.

Reply:
I like `ERROR` better. It gives a better upgrade path for users. Instead of getting:

```
Cannot find symbol 'ensureNeverFrozen'
```

They will get:

```
With the new Memory Manager this method is no longer needed
```

This leads me to realize that keeping binary compatibility here is a gimmick because any lib that would call that would most likely break under the new memory model. But it shouldn't harm either to keep these symbols around

Reply:
Agreed!